fabtests/ubertest/config.c

```diff
@@ -805,10 +820,16 @@ void fts_next(struct ft_series *series)
 		return;
 	series->cur_type = 0;
 
-	if (set->test_class[++series->cur_progress])
+	if (set->progress[++series->cur_progress])
 		return;
 	series->cur_progress = 0;
 
+	if (set->threading[++series->cur_threading]) {
+		fprintf(stderr, "threading: %d\n", series->cur_threading);
```

Comment:
Why do you print this?

Reply:
Added that when debugging. Will remove it, thanks.